### ISSUE TITLE
Logging CI failing due to too many sinks

### ISSUE THREAD

**Issue by michaelbausor** [GoogleCloudPlatform/google-cloud-java#2438]
See e.g. this build: https://travis-ci.org/GoogleCloudPlatform/google-cloud-java/builds/275116102?utm_source=email&utm_medium=notification

**Comment by neozwu** [comment]
I have manually deleted some zombie resources to unblock CI. Will investigate root cause.

**Comment by neozwu** [comment]
I'm downgrading this to P2 since it's not blocking CI at the moment. The root cause seems to be the permission setting of bigquery dataset which is used for logging sink. Will continue investigating.

**Comment by pongad** [comment]
Duplicate of #2835
